docs/deployments/compute.md

```diff
@@ -15,18 +15,22 @@ For example:
     mem: 1G
 ```
 
-CPU, GPU, and memory requests in Cortex correspond to compute resource requests in Kubernetes. In the example above, the API will only be scheduled once 1 CPU, 1 GPU, and 1G of memory are available on any instance, and it will be guaranteed to have access to those resources throughout its execution. In some cases, resource requests can be (or may default to) `Null`.
+CPU, GPU, Inf, and memory requests in Cortex correspond to compute resource requests in Kubernetes. In the example above, the API will only be scheduled once 1 CPU, 1 GPU, and 1G of memory are available on any instance, and it will be guaranteed to have access to those resources throughout its execution. In some cases, resource requests can be (or may default to) `Null`.
 
 ## CPU
 
 One unit of CPU corresponds to one virtual CPU on AWS. Fractional requests are allowed, and can be specified as a floating point number or via the "m" suffix (`0.2` and `200m` are equivalent).
 
-## Memory
-
-One unit of memory is one byte. Memory can be expressed as an integer or by using one of these suffixes: `K`, `M`, `G`, `T` (or their power-of two counterparts: `Ki`, `Mi`, `Gi`, `Ti`). For example, the following values represent roughly the same memory: `128974848`, `129e6`, `129M`, `123Mi`.
-
 ## GPU
 
 One unit of GPU corresponds to one virtual GPU. Fractional requests are not allowed.
 
 See [GPU documentation](gpus.md) for more information.
+
+## Memory
+
+One unit of memory is one byte. Memory can be expressed as an integer or by using one of these suffixes: `K`, `M`, `G`, `T` (or their power-of two counterparts: `Ki`, `Mi`, `Gi`, `Ti`). For example, the following values represent roughly the same memory: `128974848`, `129e6`, `129M`, `123Mi`.
+
+## Inf
+
+One unit of Inf corresponds to one Inferentia ASIC with 4 NeuronCores and 8GB of cache memory. Fractional requests are not allowed.
```

Comment:
Mentioning the number of available *NeuronCores* and available *cache memory* is required for the understanding of what an Inferentia ASIC offers. Otherwise, the users can waste time trying to get to understand that - the AWS documentation is not excellent either, so this way, we speed up the process of getting it. We can probably attach a hyperlink to *"Inferentia"* to make it obvious that we refer to that. Or we can say that *NeuronCores* and the *cache memory* is not the same thing as `cpu` and `mem`. What do you say?

Also, the cache memory of the Inferentia ASIC is in a way an extension of the system RAM memory.

Reply:
Specify that this isn't the same thing as `cpu` and `mem`.